docs/features/authentication.md

```diff
@@ -52,3 +74,98 @@ Auto-filling user profile depends on two conditions:
 - The REST auth module is configured for it, which is the case by default
 - Your Identity store is configured to provide profile data. See your Identity store [documentation](../backends/) on
 how to enable the feature.
+
+
+## Advanced Authentication
+The Authentication feature offers users to login to your Homeserver by using their third party identifier (3PID) registered in your Identity store.
```

Comment:
- `... feature offers users ...` -> `... feature allows users ...`
- `... to your Homeserver ...` -> `... to their Homeserver...`
- `... in your Identity store.` -> `... in an available Identity store.`